common-money.lic

```diff
@@ -29,18 +29,17 @@ module DRCM
   end
 
   def hometown_currency(hometown_name)
-    return currency = get_data('town')[hometown_name]['currency']
+    currency = get_data('town')[hometown_name]['currency']
```

Comment:
Does `currency` need to be assigned here?

Reply:
Ahh. No it doesn't.  Thanks!